src/amo/routes.js

```diff
@@ -30,7 +30,7 @@ import ServerError from './components/ErrorPage/ServerError';
 export default (
   <Route path="/:lang/:application" component={App}>
     <IndexRoute component={Home} />
-    <Route path="addon/:slug/" component={DetailPage} />
+    <Route path="addon/:slug/" component={AddonDetail} />
```

Comment:
> We don't have a generic Addon component to display bits of an Addon

I think we may use one eventually if we ever get around to DRY'ing up the Disco overlap (I swear we had an issue for this but I can't find it). Honestly, we probably won't get around to it so I don't mind renaming it.